doc/api/timers.md

```diff
@@ -61,6 +61,21 @@ timer is active. Each of the `Timeout` objects returned by these functions
 export both `timeout.ref()` and `timeout.unref()` functions that can be used to
 control this default behavior.
 
+### timeout.refresh()
+<!-- YAML
+added: REPLACEME
+-->
+
+Sets the timer's start time to the current time, and reschedules the timer to
+call its callback at the previously specified duration adjusted to the current
+time. This is useful for refreshing a timeout without allocating a new
```

Comment:
Nit/question: Should `timeout` be `timer` in this sentence? Basically, if it refers to the same thing as `timer` in the previous sentence, let's pick one word and stick with it. If it refers to something different, and the definitions are not clear elsewhere in the doc, let's add them?